clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1181,8 +1181,8 @@ public void handle(OffsetCommitResponse commitResponse, RequestFuture<Void> futu
 
         log.debug("Fetching committed offsets for partitions: {}", partitions);
         // construct the request
-        OffsetFetchRequest.Builder requestBuilder = new OffsetFetchRequest.Builder(this.rebalanceConfig.groupId,
-                new ArrayList<>(partitions));
+        OffsetFetchRequest.Builder requestBuilder =
+            new OffsetFetchRequest.Builder(this.rebalanceConfig.groupId, false, new ArrayList<>(partitions));
```

Comment:
How are you planning on setting this flag? Does the application need to know whether fetched offsets have taken into account ongoing transactions?

Reply:
My question is mostly about compatibility. Suppose we are connecting to an older broker. Will we silently revert to an older version of the protocol or will we raise an error to the user? To what extent does the user need to know that the fetched offsets reflect all pending transactions?

Reply:
I'm removing the exception thrown with just a warning.